pkgs/applications/graphics/pixelnuke/default.nix

```
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchFromGitHub, libevent, glew, glfw, ... }:
+
+stdenv.mkDerivation {
+
+  pname = "pixelnuke";
+
+  version = "2019-05-19";
+
+  src = fetchFromGitHub {
+    owner = "defnull";
+    repo = "pixelflut";
+    rev = "3458157a242ba1789de7ce308480f4e1cbacc916";
+    sha256 = "03dp0p00chy00njl4w02ahxqiwqpjsrvwg8j4yi4dgckkc3gbh40";
+  };
+
+  buildInputs = [ libevent glew glfw ];
+  buildPhase = ''
+    cd pixelnuke
+    make
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ./pixelnuke $out/bin/
+  '';
+
+  meta = with lib; {
+    description = "Pixelflut: Multiplayer canvas";
```

Doesn't fit [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/b5b66b1c80409e6e86dd9a3623f98edc4ece8020/.github/CONTRIBUTING.md):

> - `meta.description` should:
> …
>   - **Not start with the package name.**

`pixelnuke` is only one of three server implementations in the `pixelflut` repository, so I think I'd also prefer a mention of this in the description.

```suggestion
    description = "Multiplayer canvas (C implementation)";
```